pkg/codegen/docs/testdata/TestDecomposeDocstringDescription/lambda-description.golden

```diff
@@ -14,7 +14,7 @@ For information about Lambda and how to use it, see [What is AWS Lambda?](https:
 ### Basic Example
 
 <div>
-<pulumi-chooser type="language" options="typescript,python,go,csharp,java,yaml"></pulumi-chooser>
+<pulumi-chooser type="language" options="csharp,go,python,typescript,yaml,java"></pulumi-chooser>
```

Comment:
Reordering is not an issue. The `pulumi-chooser` component has an internal list of languages and just filters it with this list here.
So all code chooser still render the same